subsys/usb/class/Kconfig.cdc

```diff
@@ -53,6 +53,12 @@ config CDC_ACM_DTE_RATE_CALLBACK_SUPPORT
 	  by Arduino style programmers to reset the device into the
 	  bootloader.
 
+config USB_UART_CONSOLE
+	bool "Use CDC ACM UART as backend for console, shell, or logging"
+	help
+	  This option is only used to enable relevant options in backend
+	  configuration (if any), and for backward compatibility.
```

Comment:
No, it is in use and is investigated, see commit message:
"Kconfig option USB_UART_CONSOLE does not have any dependencies
on console drivers, but is used to select backend specific
Kconfig option in subsys/shell/backends/Kconfig.backends.
Keep the name of the option USB_UART_CONSOLE but move it
to CDC ACM configuration and update description."

Reply:
Commit was reworked. I will remove Kconfig option USB_UART_CONSOLE and dependency in subsys/shell/backends/Kconfig.backends in subsequent PR because MCUboot needs to be adapted as well.